charts/kubewarden-defaults/Chart.yaml

```
@@ -35,7 +35,7 @@ annotations:
   catalog.cattle.io/os: linux # this means linux only, other choice here is "windows". For charts that support both, don't add this annotation
   # optional ones:
   catalog.cattle.io/hidden: true # Hide specific charts. Only use on CRD charts.
-  catalog.cattle.io/auto-install: kubewarden-crds=1.4.2
+  catalog.cattle.io/auto-install: kubewarden-crds=1.4.3
   catalog.cattle.io/upstream-version: 1.8.0
```

```suggestion
  catalog.cattle.io/upstream-version: 1.8.1
```

Missing bumping the upstream-version, if we bump `version`.